spec/README.mediawiki

```diff
@@ -831,18 +831,20 @@ As part of the order, the client will submit a list of ''UTXO objects''.
 |-
 | vout      || int    || the transaction output index
 |-
-| pubkey    || string || hex-encoded public key
+| pubkeys   || [string] || array of hex-encoded pubkeys which satisfy the pubkey script
```

Comment:
`[string]` gets the point across, but I feel like we should stick to a Go-like notation `[]string`